src/main/java/org/opentripplanner/graph_builder/impl/shapefile/NihShapefileStreetGraphBuilderImpl.java

```diff
@@ -482,7 +501,48 @@ private void setProperties(SimpleFeature feature, Set<StreetEdge> edges, boolean
         LOG.info("Built option set: {}", nihOptions.toString());
 
         for (StreetEdge edge : edges) {
-            edge.setExtras(nihOptions);
+            edge.setExtraOptionFields(nihOptions);
+        }
+    }
+
+    /**
+     * Set the attributes for this feature on the matching graph edges found.
+     *
+     * @param feature Shapefile SimpleFeature with the properties to set
+     * @param edges Set of graph edges identified as belonging to the segment for this feature
+     */
+    private void setNumericProperties(SimpleFeature feature, Set<StreetEdge> edges) {
+        LOG.info("Going to set numeric properties on edges...");
+
+        // build set of converted values
+        NumericFieldSet<NihNumeric> nihNumericFieldSet = new NumericFieldSet(NihNumeric.class);
+
+        for (NihNumeric numericField : NIH_NUMERIC_FIELDS) {
+            int val = 0;
+
+            LOG.info("Going to convert field {}", numericField.getFieldName());
+
+            SimpleFeatureConverter<Integer> converter = numericConverters.get(numericField);
+
+            if (converter == null) {
+                LOG.warn("Converter for field {} not found!", numericField.getFieldName());
+            }
+
+            val = converter.convert(feature);
```

Comment:
Oh.  Leftover from debugging weirdness in source data (converter was never null.)